styleguide/source/_patterns/01-molecules/04-forms/04-in-place-editor/_in-place-editor.scss

```diff
@@ -0,0 +1,95 @@
+// Styling for the IPE when viewed in the AMA theme.
+
+  .contextual-region {
+    &:hover div.ipe-actions {
+      background: #f9f9f9;
+    }
+  }
+
+  div.ipe-actions {
+    flex-flow: row wrap;
+    margin-bottom: $gutter/2;
+    padding: 0 $gutter/2;
+    border-bottom: dotted 1px #cecece;
+  }
+
+  div.ipe-actions h5 {
+    flex-basis: 100%;
+    margin-bottom: 0;
+    padding: 8px 5px 0;
+    text-transform: none;
+  }
+
+  .ipe-actions ul.ipe-action-list {
```

Comment:
Could these all have a consistent set of selectors? Why are some `div.ipe-actions` and some just `.ipe-actions`? If one needs `.ama-theme` shouldn't they all? If not, please provide some comments to indicate what's going on with these.